indico/modules/auth/controllers.py

```diff
@@ -56,6 +56,7 @@ class RHLogin(RH):
     # CSRF token as Indico forms use. But since the form has its own
     # CSRF check anyway disabling the global check is perfectly fine.
     CSRF_ENABLED = False
+    REDIRECT_EXEMPT = ["ms-office", "Microsoft Office"]
```

Comment:
I'd put an empty line before this (so it's separated from the documented csrf block ) or even just keep the list below inside the method since it's never meant to be overridden anyway